### PR TITLE
Disable featured game websites on desktop, but not all MOWs (bug 1196359).

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @ngokevin

### CODE REVIEW

**Comment by mstriemer** [comment]
Tests?

**Comment by ngokevin** [comment]
r+, it's a hack fix and we can address the proper solution later, that the capabilities check didn't work

**Comment by mstriemer** [comment]
Websites aren't affected by the platform selector then?

File: src/media/js/apps.js [hunk 1]
```diff
@@ -147,7 +147,8 @@ define('apps',
 
             if (settings.gametimeEnabled &&
                 app.isWebsite &&
-                app.device_types.indexOf(capabilities.os.type) === -1) {
+                app.device_types.indexOf(capabilities.os.type) === -1 &&
+                app.keywords.indexOf('featured-game') !== -1) {
               // Don't show desktop website games on mobile.
```
Comment (spasovski) [inline src/media/js/apps.js]: I think this comment is a bit weak. If this is disabling a button or used in a filter etc we should probably mention it.

**Pull request merged**